snap/snapcraft.yaml

```diff
@@ -1,5 +1,6 @@
 name: charm
-version: 2.2
+version: git
+version-script: scripts/git-version
```

Comment:
These `version` and `version-script` bits are foreign to me, but reading this gives me some confidence that you're doing it right:

https://forum.snapcraft.io/t/snapcraft-version-scripts/106/44